### PR TITLE
[14.0][MIG] stock_dangerous_goods

### PR DESCRIPTION
**Pull request by mmequignon**
Depends on:
- [x] <s> https://github.com/OCA/community-data-files/pull/109 </s>
- [x] https://github.com/OCA/community-data-files/pull/137

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+odoo14-addon-l10n_eu_product_adr_dangerous_goods @ git+https://github.com/OCA/community-data-files@refs/pull/118/head#subdirectory=setup/l10n_eu_product_adr_dangerous_goods
```
Comment (sebalix) [inline test-requirements.txt]: To put in a separate TMP commit to warn reviewers that it should be removed before merging

Comment (sebalix) [reply test-requirements.txt]: And add also `l10n_eu_product_adr` the same way to get tests working.

Comment (mmequignon) [reply test-requirements.txt]: done

**Comment by simahawk** [comment]
Base PR is getting merged :tada:

**Comment by pedrobaeza** [comment]
Please avoid plurals on module name.

**Comment by rousseldenis** [comment]
> Please avoid plurals on module name.

@pedrobaeza @mmequignon Can stay as is as 'goods' is unvariable and don't exist at singular form

**Comment by pedrobaeza** [comment]
Ok, didn't know

**Comment by mmequignon** [comment]
> > Please avoid plurals on module name.
> 
> @pedrobaeza @mmequignon Can stay as is as 'goods' is unvariable and don't exist at singular form

Actually, it looks like "goods" is the plural of "good"
https://dictionary.cambridge.org/fr/dictionnaire/anglais/goods
https://dictionary.cambridge.org/fr/dictionnaire/anglais/good

**Comment by rousseldenis** [comment]
> > > Please avoid plurals on module name.
> > 
> > @pedrobaeza @mmequignon Can stay as is as 'goods' is unvariable and don't exist at singular form
> 
> Actually, it looks like "goods" is the plural of "good"
> https://dictionary.cambridge.org/fr/dictionnaire/anglais/goods
> https://dictionary.cambridge.org/fr/dictionnaire/anglais/good

Hehe didn't know too. Never used it like that

**Comment by mmequignon** [comment]
> > > > Please avoid plurals on module name.
> > > 
> > > 
> > > @pedrobaeza @mmequignon Can stay as is as 'goods' is unvariable and don't exist at singular form
> > 
> > 
> > Actually, it looks like "goods" is the plural of "good"
> > https://dictionary.cambridge.org/fr/dictionnaire/anglais/goods
> > https://dictionary.cambridge.org/fr/dictionnaire/anglais/good
> 
> Hehe didn't know too. Never used it like that

Yeah, I wasn't sure. I've discussed about this with @gurneyalex .
It looks like `dangerous good` is an alignment in dungeons and dragons.

**Comment by bosd** [comment]
Now the other 2 have merged. This one is next??

**Comment by rousseldenis** [comment]
@mmequignon Still WIP or ready?

**Comment by mmequignon** [comment]
> @mmequignon Still WIP or ready?

just rebased, this is ready :)

File: stock_dangerous_goods/models/product_product.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    is_lq_product = fields.Boolean(
```
Comment (rousseldenis) [inline stock_dangerous_goods/models/product_product.py:10]: I'd prefer explicit names than abbreviations. But nitpicking

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by simahawk — APPROVED** [review]

File: stock_dangerous_goods/models/product_product.py (hunk 2 above)
Comment (simahawk) [reply stock_dangerous_goods/models/product_product.py:10]: we could consider this for v16

**Pull request merged**